birdhouse/scripts/logging.include.sh

```diff
@@ -58,7 +58,14 @@ export LOG_CRITICAL="${REG_BG_BOLD}CRITICAL${NORMAL}: "  # to report misuse of f
 log_dest() {
     level=$1
     option=$2
-    read log_line || true
+    log_line=
+    while IFS= read -r line; do
```

Comment:
`read -r`, the `-r` is to not process `\n` inside the line if someone use it?

And not need for `|| true` ?

Reply:
Yes, we want escaped characters to be interpreted when they're printed, not when they're read.

No need for '|| true' since the while loop returns 0